lib/matplotlib/backends/backend_pdf.py

```diff
@@ -992,8 +985,12 @@ def _embedTeXFont(self, dvifont):
 
         # Reduce the font to only the glyphs used in the document, get the encoding
         # for that subset, and compute various properties based on the encoding.
-        charmap = self._character_tracker.used[(dvifont.fname, 0)]
-        chars = frozenset(charmap.keys())
+        charmap = self._character_tracker.used[dvifont.fname][0]
+        chars = {
+            # DVI fonts always map single glyph to single character.
```

Comment:
I guess that should be "type 1 fonts" in the comment (as future xe/luatex support will mean dvi will also be able to embed ttf).